css/styles.css

```diff
@@ -67,8 +67,9 @@ ul li {
 
 #about-me > ul > li > ul > li > ul > li {
   background-color: white;
-  padding: 8px;
+  padding: 8px 3px;
   border-radius: 7px;
+  font-size: 12px;
```

Comment:
I appreciate the effort you've put into your project! I wanted to provide some suggestions to further enhance it based on the template and requirements:

- [ ] Could you please take a look at the font-size of your navbar links? They appear to be slightly larger than what's shown in the template. Adjusting them to match the template's size would be great.

- [ ] It would be beneficial to review the position and padding of your main section. Adding a bit more padding could improve its visual appearance and make it more aligned with the template.

- [ ] I also noticed that the icons you've used seem to have more spacing between them compared to the template. It might be worth considering reducing the spacing to match the template's style.

   ### Figma design:
   ![image](https://github.com/flov3rh/portfolio/assets/107952724/3368f73d-9770-49e0-afd1-6b206b5762ab)

   ### Your design:
   ![image](https://github.com/flov3rh/portfolio/assets/107952724/64bb3b2e-ec3f-4ed8-aa8e-0179bb61983d)